apps/emqx_bridge_cassandra/src/emqx_bridge_cassandra_connector.erl

```diff
@@ -132,6 +132,7 @@ on_start(
                 []
         end,
     State = parse_prepare_cql(Config),
+    ok = emqx_resource:allocate_resource(InstId, pool_name, InstId),
```

Comment:
the pool name is constructed from InstId, so there should be no need to allocate resource ?